openjpa-jdbc/src/main/java/org/apache/openjpa/jdbc/schema/ForeignKey.java

```diff
@@ -810,58 +795,58 @@ public DBIdentifier loadIdentifierFromDB(DBDictionary dbdict, Connection conn) {
                 boolean addFK = false;
                 ForeignKey fkTemp = localtable.getForeignKey(
-                        fk.getIdentifier());
-                if (fkTemp == null) {
-                    addFK = true;
+                    fks[i].getIdentifier());
+                if( fkTemp == null) {
+                    addFK=true;
                     fkTemp = localtable.addForeignKey(
-                            fk.getIdentifier());
-                    fkTemp.setDeferred(fk.isDeferred());
-                    fkTemp.setDeleteAction(fk.getDeleteAction());
-                }
-                if (fk.getColumns() == null || fk.getColumns().length == 0) {
-                    // Singular column foreign key
-                    if (!fkTemp.containsColumn(
-                            localtable.getColumn(fk.getColumnIdentifier())))
-                        fkTemp.join(localtable.getColumn(fk.getColumnIdentifier()),
-                                pkTable.getColumn(fk.getPrimaryKeyColumnIdentifier()));
+                        fks[i].getIdentifier());
+                    fkTemp.setDeferred(fks[i].isDeferred());
+                    fkTemp.setDeleteAction(fks[i].getDeleteAction());
                 }
-                else {
+                if (fks[i].getColumns() == null || fks[i].getColumns().length == 0) {
+                    // Singular column foreign key 
+                    if( ! fkTemp.containsColumn(
+                        localtable.getColumn(fks[i].getColumnIdentifier())))
+                    fkTemp.join(localtable.getColumn(fks[i].getColumnIdentifier()), 
+                        pkTable.getColumn(fks[i].getPrimaryKeyColumnIdentifier()));
+                } else {
                     // Add the multi-column foreign key, joining local and pk columns in
                     // the temporary key
-                    Column[] locCols = fk.getColumns();
-                    Column[] pkCols = fk.getPrimaryKeyColumns();
+                    Column[] locCols = fks[i].getColumns();
+                    Column[] pkCols = fks[i].getPrimaryKeyColumns();
                     // Column counts must match
                     if (locCols != null && pkCols != null &&
-                            locCols.length != pkCols.length) {
+                        locCols.length != pkCols.length) {
                         Log log = dbdict.getLog();
                         if (log.isTraceEnabled()) {
                             log.trace(_loc.get("fk-column-mismatch"));
                         }
                     }
                     for (int j = 0; j < locCols.length; j++) {
-                        if (!fkTemp.containsColumn(
-                                localtable.getColumn(locCols[j].getIdentifier()))) {
-                            fkTemp.join(localtable.getColumn(locCols[j].getIdentifier()),
-                                    pkTable.getColumn(pkCols[j].getIdentifier()));
+                        if( ! fkTemp.containsColumn(
```

Comment:
`if (...)` is used in the code

not `if( ...)`

could you please revert this change?